docs/content/getting-started/code-generation.md

```diff
@@ -213,76 +209,67 @@ joined with templates and written to files
+pick the data and schema with flags. This approach can be helpful when we don't have control over the input data or if it comes in a data format.
 
 {{<codePane2
-	title1="types.go" file1="code/getting-started/code-generation/typesInput.go" lang1="go"
-	title2="> terminal" file2="code/getting-started/code-generation/out/typesInput.go" lang2="text"
+  title1= "types.go" file1= "code/getting-started/code-generation/typesInput.go" lang1= "go"
+  title2="> terminal" file2= "code/getting-started/code-generation/out/typesInput.go" lang2= "text"
 >}}
 
 
 ### Partial Templates
 
 Partial templates are fragments
-which are used in other templates.
-You can capture repeated sections
+that are used in other templates.
+Unlike regular templates, these do not map to an output file. You can capture repeated sections
 like the fields to a struct or the arguments to a function.
-Unlike regular templates, these do not map to an output file.
-Partials can also invoke other partials,
-which makes them ideal for breaking up
-your templates into logical components.
+
+
+Additionally, partials can invoke other partials, which is helpful for modularizing templates into logical components.
 
 There are two ways to define and use partial templates:
 
 - Use the `{{ define "name" }}` syntax in a regular template
 - User the `-P` to load them from a file
 
-Let's extract field generation into its own template, where we _could_ make it complex.
-We won't here, but an example is struct tags for our Go fields.
-We can also use template helpers in the output filepath.
+For example, we _could_ extract the generation of fields into its template. We won't here, but as an example could include complex tasks like struct tags for Go fields. 
 
 {{<codeInner title="> terminal" lang="sh">}}
 $ hof gen data.cue schema.cue -P field.go -T types.go -O out/
 {{</codeInner>}}
 
 {{<codePane3
-	title1="types.go" file1="code/getting-started/code-generation/typeP.go" lang1="go"
-	title2="field.go" file2="code/getting-started/code-generation/field.go" lang2="go"
-	title3="out/types.go" file3="code/getting-started/code-generation/out/types.go" lang3="text"
+  title1="types.go" file1="code/getting-started/code-generation/typeP.go" lang1="go"
+  title2= "field.go" file2= "code/getting-started/code-generation/field.go" lang2= "go"
+  title3= "out/types.go" file3= "code/getting-started/code-generation/out/types.go" lang3= "text"
 >}}
 
 
 ### Repeated Templates
 
-We just saw how to loop over data and apply a template fragment.
-We can also render with __repeated templates__, which are processed
-for each element of an iterable (list or struct fields),
-and which also write a file for each element.
-
-Use `[]` to render a file for each element in the input to a `-T` flag.
+In addition to looping over data and applying a template fragment, as mentioned in the previous example, you can use __repeated templates__ to write a file for each element in an iterable such as a list or struct field. 
```

Comment:
In addition to looping over data and applying a template fragment, you can use __repeated templates__ to write a file for each element in an iterable such as a list or struct field.